src/main/java/cloud/eppo/BaseEppoClient.java

```diff
@@ -22,37 +22,46 @@ public class BaseEppoClient {
   private final ObjectMapper mapper =
       new ObjectMapper()
           .registerModule(EppoModule.eppoModule()); // TODO: is this the best place for this?
-  private static final String DEFAULT_HOST = "https://fscdn.eppo.cloud";
-  private static final boolean DEFAULT_IS_GRACEFUL_MODE = true;
+
+  protected static final String DEFAULT_HOST = "https://fscdn.eppo.cloud";
+  protected final ConfigurationRequestor requestor;
 
   private final ConfigurationStore configurationStore;
-  private final ConfigurationRequestor requestor;
   private final AssignmentLogger assignmentLogger;
   private final BanditLogger banditLogger;
   private final String sdkName;
   private final String sdkVersion;
   private final boolean isConfigObfuscated;
   private boolean isGracefulMode;
 
-  private static BaseEppoClient instance;
-
   // Fields useful for testing in situations where we want to mock the http client or configuration
   // store (accessed via reflection)
   /** @noinspection FieldMayBeFinal */
   private static EppoHttpClient httpClientOverride = null;
 
-  private BaseEppoClient(
+  public BaseEppoClient(
```

Comment:
Should this be protected so it's only callable by subclasses?

Reply:
Good idea! I had it public as it's used by automated tests, but forgot it will still have _package_ access. Will change!

Reply:
+1 to making this protected